sql/c-other-functions.sql

```diff
@@ -1,7 +1,9 @@
-CREATE FUNCTION following(int) RETURNS int[] AS $$
+
+CREATE FUNCTION following(int, int) RETURNS int[] AS $$
     SELECT array_append(array_agg(followee_id), $1)
       FROM follows
      WHERE follower_id = $1
+     LIMIT $2
```

Comment:
I don't think putting the limit outside of the array construction works, also if we're going to have a limit we need to pick a stable order, I guess followed.

Something like this? Untested:
```sql
WITH follows AS (
    SELECT followee_id
      FROM follows
     WHERE follower_id = $1
  ORDER BY followed
     LIMIT $2
) SELECT array_append(array_agg(SELECT * FROM follows), $1)
```